bears/julia/JuliaLintBear.py

```diff
@@ -25,7 +25,7 @@ class JuliaLintBear:
     on the analysis provided.
     """
     LANGUAGES = {'Julia'}
-    REQUIREMENTS = {DistributionRequirement(apt_get='julia')}
+    REQUIREMENTS = {JuliaRequirement(apt_get='julia')}
```

Comment:
@prakashdanish No need of `apt_get` in the JuliaRequirement. The arguments should be like `('package name', 'package version')`